plugins/cisco_firepower_management_center/Dockerfile

```diff
@@ -13,6 +13,7 @@ RUN if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
 ADD . /python/src
 
 RUN	python setup.py build && python setup.py install
+RUN pip uninstall ipaddress -y
```

Comment:
doing a quick check locally we do use the lib within the plugin, do we maybe need to pin a vuln free version of this lib in requirements?

Reply:
The latest version of this module is considered vulnerable by Snyk (1.0.23)
https://pypi.org/project/ipaddress/

The vulnerability got fixed in Python 3.9.5 but currently the sdk points to 3.9.18
https://bugs.python.org/issue36384#msg392423